NickvisionMoney.GNOME/Blueprints/account_settings_dialog.blp

```diff
@@ -57,7 +57,6 @@ Adw.Window _root {
 
                 Adw.ComboRow _accountTypeRow {
                   title: _("Account Type");
-                  subtitle: _("This is only a useful label that doesn't affect how the app works.");
```

Comment:
Because now that we have the New Account setup dialog, this description is exposed there ans the user can learn that it's just for cosmetic purposes...now real purpose to have it in the Settings dialog itself